get_reference_housenumbers.py

```diff
@@ -70,6 +73,16 @@ def getStreetURL(street, prefix):
         district = "xi"
     elif suffix == "-spanyolret":
         district = "xi"
+    elif suffix == "-csilleberc":
+        district = "xii"
+    elif suffix == "-dobogo":
+        district = "xi"
+    elif suffix == "-kelenfold":
+        district = "xi"
+    elif suffix == "-orsod":
+        district = "xi"
+    elif suffix == "-szechenyihegy":
+        district = "xii"
```

Comment:
Lassan ezt is ki kellene tenni egy map-ba vagy külön fájlba.
`R0912: Too many branches (17/12) (too-many-branches)`

Reply:
a relation mellé nem tudjuk tenni numerikusan, akkor később fel lehetne használni, mert az új referenciában (2019) már numerikusan van
nem tudom yaml-ul hogy van de ilyen kellene

farkasret: {id:2787964, megyekod: "01", telepuleskod: "011"}
...
es vmi ilyesmi a phytonban, ami tudna kezelni a regi es uj ref-et is
def get_telepules_2016(megyekod, telepuleskod,) 
..
if megyekod == "01":
  return  "/budapest%20" + arab_to_romai(telepuleskod) + ".ker./"

es majd vmikor az uj referenciara
def get_telepules_2019(megyekod, telepuleskod) 
   return "&_wardsearch_WAR_nvinvrportlet_megyeKod" + megyekod + "&_wardsearch_WAR_nvinvrportlet_telepulesKod" + telepuleskod

Reply:
Jaja, megcsinálom. Menjen be ez így aztán javítom.

Reply:
Helyrepofoztam a Travist meg a vmiklos.pythonanywhere.com/osm-et is befrissítettem.